attributes/default.rb

```diff
@@ -1,2 +1,3 @@
 default['scrutinizer-whitelist']['source-url'] = 'https://scrutinizer-ci.com/api/meta'
 default['scrutinizer-whitelist']['priority'] = '05'
+default['scrutinizer-whitelist']['ports'] = [22]
```

Comment:
note the recipes of the cookbook didn't need to be included, but this helped drive it from one config value